scripts/json2xml/script.js

```diff
@@ -0,0 +1,47 @@
+import pkg from "jstoxml";
+const { toXML } = pkg;
+import fs from "fs";
+import path from "path";
+
+// Get the JSON file path from command-line arguments
+const jsonFilePath = process.argv[2];
+
+if (!jsonFilePath) {
+  console.error("Please provide a JSON file path as an argument.");
+  process.exit(1);
+}
+
+// Read the JSON file
+fs.readFile(jsonFilePath, "utf8", (err, data) => {
```

Comment:
Instead of using the old callback `readFile`, you should update to the promise based version with `async` `await`, see https://nodejs.org/api/fs.html#fspromisesreadfilepath-options.